### DISCUSSION TITLE
Add team/teams ownership information in the repository main page

### DISCUSSION THREAD

**Discussion by Avivhdr** [community/community#9021] [Repositories]
Hi Github team 👋🏼 
In our organization, we have 1000+ repositories with dozens of teams.
Each repository is owned by one team or more.

**The problem:**
If I'm viewing a repository and want to know which team owns that repository, I can't (AFAIK).

I can go to my organization's main page on Github, choose the `Teams` tab, and see the list of all the existing teams.
That doesn't help though, because It doesn't take me closer to knowing who is the team that owned the repository I was viewing.
![image](https://user-images.githubusercontent.com/23265867/146597786-94c2fad3-40f5-4520-b862-95d40834e236.png)

**I would suggest 2 possible solutions:**
1. The `Teams` tab has a search input that allows you to search by team name. Add the option to search for repository names as well. the team/s that owns the repository you were searching, will be the ones that would show up in the search result. 
2. A better solution IMO - Add a `Team Ownership` section in the information pane on the right side of the file-system (`<>Code` tab). Illustration 👇🏼 
![image](https://user-images.githubusercontent.com/23265867/146596874-2b2ce903-9093-439c-851e-e5a1149b9dce.png)

Thank you very much 🙏🏼

**Comment by alexef** [comment]
Also very interested in this feature.

**Comment by dschaefer2** [comment]
Me too? Funny this isn't a thing. How do you know what team to ask to join to get access rights on a repo?

**Comment by aag193** [comment]
I am long awaiting this feature. easier to sort/track things for the team

**Comment by KevinPartingtonTR** [comment]
This is biting us in the `tr` organization. We have a "Using GitHub" chat in Microsoft Teams and it very frequently receives requests to be added to teams (or added as maintainer/admin of a repo). If teams associated with a repository were made visible on the repository page, we could educate people to reach out to members of those teams for permission changes.
